scrapy/crawler.py

```diff
@@ -123,17 +161,43 @@ def _setup_crawler_logging(self, crawler):
             crawler.signals.connect(log_observer.stop, signals.engine_stopped)
 
     def stop(self):
+        """
+        Stops simultaneously all the crawling jobs taking place.
+
+        Returns a deferred that is fired when they all have ended.
+        """
         return defer.DeferredList([c.stop() for c in list(self.crawlers)])
 
     @defer.inlineCallbacks
     def join(self):
-        """Wait for all managed crawlers to complete"""
+        """
+        join()
```

Comment:
The first line is there so Sphinx can get the right signature of `join()`, because of the decorator it's computed as `join(*args, **kwargs)` otherwise.